Tools/astyle/pre-commit

```diff
@@ -49,6 +49,7 @@ fi
 # If there are whitespace errors, print the offending file names and fail.
 git diff-index --check --cached $against --
 if [ $? -ne 0 ]; then
+	echo "Pre-commit style error: Whitespace issues"
```

Comment:
Style error: you're mixing tabs and spaces :smile:

Reply:
Good point 👍 , vscode detected the prevalently used indentation method in the file and everything I did was with tabs. Now I put a commit to change the rest of the file. It's consistent with the other shell scripts I looked at.